seqr/views/utils/test_utils.py

```diff
@@ -914,22 +914,24 @@ def _get_list_param(call, param):
             'gnomad_genomes': {'af': 0.007},
         },
         'genotypes': {
-            'NA19675': {
+            'NA19675': [{
```

Comment:
i would change these back since they should not be changed when theres only one sample type. We actually import  the fixture variants from hail_search/test_utils in a lot of our seqr app tests, so I feel like to move would be to update those tests to also include a mutli-sample variant from the hail search tests as well

Reply:
so I want to just add a mutli-sample variant to the expected api response in `VariantSearchAPITest` - do you know if I'll have to update any of the django fitxures?